packages/odyssey-react-mui/src/Infobox.tsx

```diff
@@ -0,0 +1,52 @@
+/*!
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { AlertColor } from "@mui/material";
+import { memo, ReactNode } from "react";
+import { Alert, AlertTitle } from ".";
+
+export interface InfoboxProps {
+  /**
+   * The contents of the alert
+   */
+  children: ReactNode;
+  /**
+   * Determine the color and icon of the alert
+   */
+  severity?: AlertColor;
+  /**
+   * Sets the ARIA role of the alert
+   * ("status" for something that dynamically updates, "alert" for errors, null for something
+   * unchanging)
+   */
+  role?: "status" | "alert";
+  /**
+   * The title of the alert
+   */
+  title?: string;
+}
+
+const Infobox = ({
+  children,
+  severity = "info",
```

Comment:
Do we want a default `severity`? Seems like no severity at all and then requiring it is better because there's never a situation where it doesn't make sense to know. As in, they should have intent behind using an `Infobox`.